CODEX_SETUP.md

```
@@ -0,0 +1,228 @@
+
+```bash
+cd /workspace/ComfyUI
+git pull
+pip install -r requirements.txt
+```
+
+---
+
+## 📚 Resources
+
+- **ComfyUI Docs:** https://github.com/comfyanonymous/ComfyUI
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)